src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/Options/CollectionRuleOptionsExtensions.cs

```diff
@@ -168,6 +168,30 @@ public static CollectionRuleOptions AddLoadProfilerAction(this CollectionRuleOpt
                 });
         }
 
+        public static CollectionRuleOptions AddSetEnvironmentVariableAction(this CollectionRuleOptions options, Action<SetEnvironmentVariableOptions> configureOptions)
```

Comment:
Rather than using a fluent-style API here, just have `string name, string value` parameters and set it within the method.